src/collects/seashell/backend/dispatch.rkt

```diff
@@ -108,10 +112,21 @@
             (send-message connection `#hash((id . -4) (success . #t)
                                             (result . #hash((pid . ,pid) (test_name . ,test-name) (result . ,test-res)))))]
            [(list pid test-name "error" exit-code stderr)
+            ;; check for asan error message in "{project}/{question}/.asan.{pid}"
```

Comment:
Alright I'll change that. Just let you know that the asan message still needs to be sent back to the frontend through the "type: done" api if user runs their code without tests, and the compilation is success. (in which case the only options are to send back through "type: done", or through the last "type: stderr")